### PR TITLE
Remove Travis, use GitHub Action

### PR DESCRIPTION
**Pull request by sidvishnoi**
https://github.com/w3c/gamepad/pull/109 from non-fork branch, so we can test run before merging

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]
Nice! thanks for fixing the env stuff :)

**Comment by marcoscaceres** [comment]
Marked as non substantive for IPR from ash-nazg.

File: .github/workflows/pr-push.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+      - name: Do spec validation
+        env:
+          GH_USER: ${{ secrets.GH_USER }}
+          GH_TOKEN: ${{ secrets.GH_TOKEN }}
+        run: npx respec-validator --gh-user=$GH_USER --gh-token=$GH_TOKEN index.html
```
Comment (marcoscaceres) [inline .github/workflows/pr-push.yml]: I think you need to install ReSpec unfortunately.

Comment (sidvishnoi) [reply .github/workflows/pr-push.yml]: respec is a dependency of respec-validator, so it should've been installed with npx, right?
let me try locally also

Comment (sidvishnoi) [reply .github/workflows/pr-push.yml]: Interesting

Comment (sidvishnoi) [reply .github/workflows/pr-push.yml]: Glad we did a test run

Comment (marcoscaceres) [reply .github/workflows/pr-push.yml]: Pfff... whatever. :shipit:

**Pull request merged**